samples.http

```diff
@@ -0,0 +1,63 @@
+@baseUrl = http://localhost:3001/api
```

Comment:
problematic to hardcode the port of the local server here, better to use the environment variable directly:
{{$processEnv [%]envVarName}} --> https://github.com/Huachao/vscode-restclient
```suggestion
@baseUrl = http://localhost:{{$processEnv [%]PORT}}/api
```